### PR TITLE
Dev/fpla 2288 : Remove Feature toggle for bulk user assignment 

### PR DESCRIPTION
**Pull request by nkoranne**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2288

### Change description ###
Removal of feature toggle for bulk user assignments 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java [hunk 1]
```diff
@@ -415,30 +311,18 @@ private void verifyTaskListUpdated(CaseDetails caseDetails) {
...
-            throw new FeignException.NotFound("", request, new byte[]{});
-        });
-    }
-
     private void givenPRDWillReturn(List<String> userIds) {
         givenPRDWillAnswer(invocation -> organisation(userIds));
     }
 
-    private void givenPRDWillAnswer(Answer... answers) {
+    private void givenPRDWillAnswer(Answer<?>... answers) {
```
Comment (tomaszpowroznik) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java:318]: this should be not used now. It is used in single place, which can be written much more cleaner. Was like this to record sequence of responses, now as there is not retries it is not needed

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java [hunk 2]
```diff
@@ -55,10 +51,6 @@ public boolean isSendLAEmailsToSolicitorEnabled(String localAuthorityName) {
             createLDUser(Map.of(LOCAL_AUTHORITY_NAME_KEY, LDValue.of(localAuthorityName))), false);
     }
 
-    public boolean isUploadDraftCMOEnabled() {
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java:58]: not really belongs to this ticket, but should be ok

**Review by tomaszpowroznik — APPROVED** [review]

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java (hunk 1 above)
Comment (nkoranne) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java]: as discussed , removed givenPRDWillReturn and replaced givenPRDWillReturn  with your suggestion : 
private void givenPRDWillReturn(List<String> userIds) {
    given(organisationApi.findUsersInOrganisation(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN, Status.ACTIVE, false)).willReturn(organisation(userIds));
}

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java (hunk 2 above)
Comment (nkoranne) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java:58]: will keep it

**Pull request merged**